_templates/component/new/update-top-level-scss.ejs.t

```diff
@@ -0,0 +1,7 @@
+---
+inject: true
+to: _index.scss
+append: true
+---
+
+@import './src/components/<%= h.inflection.dasherize(component_name) %>/_index.scss';
```

Comment:
The Sass library ticket is very likely going to affect this - and this will need changed depending on the outcome of that ticket

Reply:
gotcha